lib/evmone/baseline_execution.cpp

```diff
@@ -335,19 +335,21 @@ evmc_result execute(evmc_vm* c_vm, const evmc_host_interface* host, evmc_host_co
 {
     auto vm = static_cast<VM*>(c_vm);
     const bytes_view container{code, code_size};
+    const auto eof_enabled = rev >= EVMC_PRAGUE;
```

Comment:
Can you use `REV_EOF1` here?

Reply:
Added as `instr::REV_EOF1` but revision handling can be improved further. I wonder if we can make the revision opaque for EVMC.